style/style.css

```diff
@@ -41,6 +43,59 @@ body {
     background-color: green;
 }
 
+/* DROPDOWN */
+
+.dropdown {
```

Comment:
No separate style file for component?

Reply:
I'm working on it. This will be handled in version 2.0